Assets/LeapMotion/Scripts/Utils/Hands.cs

```diff
@@ -0,0 +1,169 @@
+﻿using UnityEngine;
+using System.Collections.Generic;
+using UnityEngine.SceneManagement;
+
+namespace Leap.Unity {
+
+  public static class Hands {
+
+    private static LeapServiceProvider s_provider;
+    private static GameObject s_leapRig;
+
+    static Hands() {
+      InitStatic();
+      SceneManager.activeSceneChanged += InitStaticOnNewScene;
+    }
+
+    private static void InitStaticOnNewScene(Scene unused, Scene unused2) {
+      InitStatic();
+    }
+
+    private static void InitStatic() {
+      s_provider = GameObject.FindObjectOfType<LeapServiceProvider>();
+      s_leapRig = s_provider.transform.parent.parent.parent.gameObject;
+    }
+
+    /// <summary>
+    /// Static convenience accessor for the Leap Rig GameObject.
+    /// </summary>
+    public static GameObject Rig {
```

Comment:
Possibly search for a camera as a parent of the provider, then get it's parent?  I think that would be at least safer than using multiple parent accessor.  If we cannot find a reliable and safe way of getting it, we should not include it here.